### ISSUE TITLE
make possible to set own defaults for /boot/bin/cubietruck.bin after upgrade

### ISSUE THREAD

**Issue by RafaelKa** [igorpecovnik/Cubietruck-Debian#53]
apt-get upgrade command overrides `/boot/bin/cubietruck.bin` 
it should be possible to set own settings in some conf file, which should be merged after apt-get upgrade.

as follows:
```bin2fex /boot/bin/cubietruck.bin /boot/script.fex
# step to set properties from /boot/script.fex.custom in /boot/script.fex
fex2bin /boot/script.fex /boot/bin/cubietruck.bin```

**Comment by igorpecovnik** [comment]
You can have your own settings. I am only updating hw defaults.

Kernel is always loading script.bin which is a symbolic link to one of the files inside /boot/bin .... add yours and relink.

**Comment by RafaelKa** [comment]
Yes, but then changed in your package settings are lost after update!
